### PR TITLE
Mbishop edits sds vs omf

### PR DESCRIPTION
**Pull request by osisoft-mbishop**
Hi Collin,

Good draft! Here are the tweaks I made to your article. 

- Replaced upfront links with a comparison table for TLDR people. 
- Minor spelling and grammar changes.
- Added a couple of headings for doc structure that our portal uses to make a mini toc.
- Replaced a few link URLs with IDs for more resiliency.
- Replaced AVEVA branding with OSIsoft. We're in state where we're maintaining two doc sets--one for OCS and one for ADH. We're merging into the OCS doc set here. When I merge these changes downstream to ADH, I'll do the rebrand then. 
- I also added this doc to the OCS toc. There's a preview available [here](https://osisoft-dev.zoominsoftware.io/bundle/ocs-mbishop-edits-sds-vs-omf/page/developer-guide/omf-dev/omf-sds-vs-omf.html).

### CODE REVIEW

File: content/developer-guide/omf-dev/omf-sds-vs-omf.md [hunk 1]
```diff
@@ -2,56 +2,63 @@
...
+Additionally, not all OMF endpoints respond with the same error codes due to asynchronous message processing, which could result in a successful HTTP status code being returned when a SDS data write would result in a client error status code. This could happen if a message adheres to the OMF specification, but is not writing to a valid container.  
 
-**Winner:** Situation dependent
+## Situation dependent factors
+
+For some factors, neither SDS nor OMF are inherently superior. 
+
+### Security
+
+Both SDS and OMF use the same authentication mechanism when connecting to OCS, so neither is inherently more secure. Always follow best practices for security as outlined in the [Identity and access management](xref:id-access-mgmt).
```
Comment (cbardini) [inline content/developer-guide/omf-dev/omf-sds-vs-omf.md:56]: I think removing "the" makes this sound more natural. Also, the link does not render properly.
```suggestion
Both SDS and OMF use the same authentication mechanism when connecting to OCS, so neither is inherently more secure. Always follow best practices for security as outlined in [Identity and access management](xref:id-access-mgmt).  
```

File: content/developer-guide/omf-dev/omf-sds-vs-omf.md [hunk 2]
```diff
@@ -2,56 +2,63 @@
...
+The APIs used in SDS are continually updated, requiring future upkeep of your applications. OMF does not require this upkeep, as it is up to the platform to interpret the messages and store the contained data appropriately.  
 
-**Winner:** OMF
+### Cross-platform
 
-## Flexibility
-SDS data writes afford greater flexibility than OMF, which may make it the clear winner for certain use cases.  
-If a requirement of your application is both reading and writing from ADH, then SDS data writes are preferable to avoid the need for interacting with two separate APIs. Using the SDS API allows for reuse of objects such as types and streams in your application.  
+While SDS data writes can only be used for OSIsoft Cloud Services (OCS) and Edge Data Store (EDS), OMF can also write to a PI Core Server through the PI Web API. This additional capability makes OMF cross-platform, which is one of the main reasons that it is used in OSIsoft adapter technology. In addition, other platforms may adopt the OMF standard to ingress data, widening the interoperability of your applications.
```
Comment (cbardini) [inline content/developer-guide/omf-dev/omf-sds-vs-omf.md:32]: Should adapter be capitalized?

**Review by cbardini — COMMENTED** [review]
Thanks for the review, Mark! I really like the comparison table and section headers.

File: content/developer-guide/omf-dev/omf-sds-vs-omf.md (hunk 2 above)
Comment (osisoft-mbishop) [reply content/developer-guide/omf-dev/omf-sds-vs-omf.md:32]: It's a little unclear if "adapter" is part of a proper noun here, which is why I got rid of the capitalization. Usually it's best to default to non-caps if you're unclear. Let me check around to see how marketing refers to adapters.

Comment (osisoft-mbishop) [reply content/developer-guide/omf-dev/omf-sds-vs-omf.md:32]: I updated this to "PI Adapters" based on precedent from this marketing page: https://www.osisoft.com/pi-system/pi-core/collection

**Pull request merged**